### PR TITLE
Fix: Add new admin to Training and Prod

### PR DESCRIPTION
**Pull request by krisrjohnson21**
We have requests to add new admin for the CharlieCard Store. This PR adds those admin to the data source for both Training and Production environments. 

Asana tasks:
- https://app.asana.com/0/1113179098808463/1202391509377031/f
- https://app.asana.com/0/1113179098808463/1202389178503658/f

### CODE REVIEW

File: data_sources/Production Senior Permissions.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ﻿Value,Admin 1,Admin 2,Admin 3,Admin 4,Admin 5,Admin 6,Admin 7,Admin 8,Admin 9,Admin 10,Admin 11,Admin 12,Admin 13,Admin 14,Admin 15
-correct,fmoonalewis@mbta.com,jho@mbta.com,jlewis3@mbta.com,jmacdonald1@mbta.com,ldesnoyer@mbta.com,lserrano@mbta.com,tdegrace@mbta.com,narmstrong@mbta.com,rshiels@mbta.com,nbowman@mbta.com,,,,,
+correct,fmoonalewis@mbta.com,jho@mbta.com,jlewis3@mbta.com,jmacdonald1@mbta.com,ldesnoyer@mbta.com,lserrano@mbta.com,tdegrace@mbta.com,narmstrong@mbta.com,rshiels@mbta.com,nbowman@mbta.com,ddottinsteele@mbta.com,obazile@mbta.com,bmccarthy2@mbta.com,,,
```
Comment (karenhawkinson) [inline data_sources/Production Senior Permissions.csv:2]:
```suggestion
correct,fmoonalewis@mbta.com,jho@mbta.com,jlewis3@mbta.com,jmacdonald1@mbta.com,ldesnoyer@mbta.com,lserrano@mbta.com,tdegrace@mbta.com,narmstrong@mbta.com,rshiels@mbta.com,nbowman@mbta.com,ddottinsteele@mbta.com,obazile@mbta.com,bmccarthy2@mbta.com,,

```

File: data_sources/Training Senior Permissions.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
 ﻿Value,Admin 1,Admin 2,Admin 3,Admin 4,Admin 5,Admin 6,Admin 7,Admin 8,Admin 9,Admin 10,Admin 11,Admin 12,Admin 13,Admin 14,Admin 15
-correct,fmoonalewis@mbta.com,jho@mbta.com,jlewis3@mbta.com,jmacdonald1@mbta.com,ldesnoyer@mbta.com,lserrano@mbta.com,tdegrace@mbta.com,narmstrong@mbta.com,rshiels@mbta.com,nbowman@mbta.com,vfranklin@mbta.com,,,,
+correct,fmoonalewis@mbta.com,jho@mbta.com,jlewis3@mbta.com,jmacdonald1@mbta.com,ldesnoyer@mbta.com,lserrano@mbta.com,tdegrace@mbta.com,narmstrong@mbta.com,rshiels@mbta.com,nbowman@mbta.com,vfranklin@mbta.com,ddottinsteele@mbta.com,obazile@mbta.com,bmccarthy2@mbta.com,,
```
Comment (karenhawkinson) [inline data_sources/Training Senior Permissions.csv:2]:
```suggestion
correct,fmoonalewis@mbta.com,jho@mbta.com,jlewis3@mbta.com,jmacdonald1@mbta.com,ldesnoyer@mbta.com,lserrano@mbta.com,tdegrace@mbta.com,narmstrong@mbta.com,rshiels@mbta.com,nbowman@mbta.com,vfranklin@mbta.com,ddottinsteele@mbta.com,obazile@mbta.com,bmccarthy2@mbta.com,

```

**Review by karenhawkinson — CHANGES_REQUESTED** [review]
@krisrjohnson21 I believe there should be one fewer comma in each file. Check "view file" to see if it populates correctly. Thanks!

**Comment by krisrjohnson21** [comment]
> @krisrjohnson21 I believe there should be one fewer comma in each file. Check "view file" to see if it populates correctly. Thanks!

Good call! It was populating correctly in SimpliGov but I didn't need these end commas.

**Review by karenhawkinson — APPROVED** [review]
:white_check_mark:

**Pull request merged**